tests/misc_env/install_prereq.py

```diff
@@ -173,6 +184,41 @@ def enable_rhel_rpms(ceph, distro_ver):
         )
 
 
+def enable_rhel_eus_rpms(ceph, distro_ver):
+    """
+    Setup cdn repositories for rhel systems
```

Comment:
Want to link to http://wiki.test.redhat.com/CEPH/SubscriptionManager here?

Reply:
The PR text also links to https://access.redhat.com/solutions/1305953, and that would be a good thing to link as well.

Reply:
I have updated the reference article in the PR, I think your wiki makes more sense.